src/mame/roland/roland_jv80.cpp

```diff
@@ -19,25 +21,30 @@ class roland_jv80_state : public driver_device
 	roland_jv80_state(const machine_config &mconfig, device_type type, const char *tag)
 		: driver_device(mconfig, type, tag)
 		, m_maincpu(*this, "maincpu")
+		, m_pcm(*this, "pcm")
 	{
 	}
 
 	void jv880(machine_config &config);
 
 private:
-	void mem_map(address_map &map);
+	void jv880_mem_map(address_map &map);
 
 	required_device<h8532_device> m_maincpu;
+	required_device<tc6116_device> m_pcm;
 };
 
 
-void roland_jv80_state::mem_map(address_map &map)
+void roland_jv80_state::jv880_mem_map(address_map &map)
 {
-	map(0x08000, 0x09fff).ram();
+	map(0x00000, 0x08000).rom().region("maincpu", 0);
```

Comment:
That should be 0x7fff and not 0x8000.  Recommend you try a mame -valid before doing a submission, it doesn't take long and avoids duh-moments.

Reply:
Hi, just pushed a fix, thanks for the tip!
I'm making my first contributions and I'm still trying to learn how all the internal works, sorry.